packages/direction-link/docs/overview.mdx

```diff
@@ -28,23 +28,21 @@ Use direction links to indicate a physical direction, such as:
 </Stack>
 ```
 
-### Buttons
+### DirectionButton
 
-Sometimes direction is needed inside a form. Buttons offer a way to direct users to the next or previous section inside a form.
+By default, the `DirectionLink` component renders a HTML anchor tag. We strongly recommend using it wherever possible as it is the most inclusive and accessible solution for users.
 
-```jsx live
+But using a link may be technically inappropriate for some single-page-app experiences, where navigating to another web page would erase all user-entered form information.
+
+`DirectionButton` offers a way to direct users through sections inside a form or single-page-app, without triggering a page navigation. It looks visually identical to a `DirectionLink`, but it renders a button element instead.
+
+```jsx
 <Stack gap={2}>
-	<DirectionButton onClick={console.log} direction="left">
+	<DirectionButton onClick={setCurrentSection(1)} direction="left">
```

Comment:
```suggestion
	<DirectionButton onClick={() => goToPreviousStep()} direction="left">
```

Reply:
I think this might be easier to understand, what are your thoughts @nathsimpson ?